api_docs/changelog.md

```diff
@@ -34,7 +34,8 @@ format used by the Zulip server that they are interacting with.
   the `subject` field in the `message` event type, `topic` field in
   the `delete_message` event type, `orig_subject` and `subject` fields
   in the `update_message` event type, `topic_name` field in the `user_topic`
-  event type will have the value `"general chat"` replacing `""` for channel messages.
+  event type, `topic` field in the `typing` event type will have the value
+  `"general chat"` replacing `""` for channel messages.
```

Comment:
I think this might have been better written with a 2-layer bulleted list. First, an outer one giving the context on the change, and then the inner one just listing the fields.

Reply:
<img width="686" alt="Screenshot 2024-12-19 at 10 24 03 PM" src="https://github.com/user-attachments/assets/560e0943-bc8f-474b-b1f2-2e69da7e3a05" />